packages/quantic/force-app/main/default/classes/ITokenProvider.cls

```diff
@@ -0,0 +1,3 @@
+global interface ITokenProvider {
```

Comment:
That seem to introduce a weird circle dependencies between the packages ? You can't push the custom-token package without this interface, but you can't push de default package without de custom-token package? Or I'm missing something?

Reply:
As you mentioned, the main package should use the sample.
So the circular dependency will not be here.